### ISSUE TITLE
Lookbehind ontaining `>` character is parsed as a named group

### ISSUE THREAD

**Issue by dvandersluis** [ammar/regexp_parser#93]
```ruby
Regexp::Lexer.lex(/(?<=>)y/)
=> [#<struct Regexp::Token type=:group, token=:named, text="(?<=>", ts=0, te=5, level=0, set_level=0, conditional_level=0>,
 #<struct Regexp::Token type=:group, token=:close, text=")", ts=5, te=6, level=0, set_level=0, conditional_level=0>,
 #<struct Regexp::Token type=:literal, token=:literal, text="y", ts=6, te=7, level=0, set_level=0, conditional_level=0>]
```

This should be a lookbehind assertion with a literal `>`.

**Comment by dvandersluis** [comment]
Thank you @jaynetics! Any chance this might be released soon?

**Comment by jaynetics** [comment]
thank you very much for the report and PR! :)

i've just released v2.9.3 with your fix.

**Comment by dvandersluis** [comment]
Thank you!
